DuoSecurity/local_settings.py-dist

```diff
@@ -1,6 +1,7 @@
 LDAP_HOST = '127.0.0.1'
 LDAP_USERNAME = ''
 LDAP_PASSWORD = ''
+GRACE_PERIOD = 10080
```

Comment:
Depending on how strapped for licenses we are, I suggest starting this out at more like 30 or 60 or even 90 days, as we might want to hold onto login history for a given user for forensic purposes longer than 7 days